### ISSUE TITLE
Internal Server Errors on paste?

### ISSUE THREAD

**Issue by beyarkay** [DarrenOfficial/dpaste#246]
## Description

Hi! I'm getting 500s when I try to paste something:

```
$ echo "some text to paste" | curl -X POST -F "expires=31536000" -F 'format=url' -F 'content=<-' https://dpaste.org/api/ > pastebin.txt
$ cat pastebin.txt

...html of the 500 error page...
```

I get the same error trying to paste on https://dpaste.org/ by just clicking the "paste snippet" button.

Let me know if I can help out! I'm using this for my project eskom-calendar so am happy to assist.

**Comment by some1ataplace** [comment]
Same problem here.

**Comment by DarrenOfficial** [comment]
It appears that the VM has ran out of storage...
![inf](https://github.com/DarrenOfficial/dpaste/assets/55406094/895846c0-7f3b-4f11-a37c-7015066fe1a1)

**Comment by DarrenOfficial** [comment]
Everything should have been sorted out by now, I've increased the storage size from 40 -> 80 I'll investigate what's causing this sudden increase in storage size shortly... because this is really weird.

![df](https://github.com/DarrenOfficial/dpaste/assets/55406094/2af428b8-a4d7-4343-bc44-d99f55163190)

**Comment by DarrenOfficial** [comment]
I've found the root cause, it was because of a misconfiguration for pgsql log statement. The default config on this server was to log all;

![pgsql](https://github.com/DarrenOfficial/dpaste/assets/55406094/3a20a643-01b6-46ac-960b-2ecaf66eabab)

https://postgresqlco.nf/doc/en/param/log_statement/

>  Recommendations
For exhaustive performance analysis on test systems, set to 'all'. Most production setups will just want to use 'ddl' to make sure to record database-altering actions, but very secure setups may want to use 'mod' or even 'all'. **Can produce a lot of log volume**.

I've purged the logs and re-configure pgsql, everything should be fine now 🙂
